src/main/java/com/selenium/course/pages/Page.java

```diff
@@ -4,12 +4,9 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-/**
- * Created by Rina Espinoza on 8/28/2015.
- */
 public class Page {
-    protected WebDriver driver;
-    protected WebDriverWait wait;
+    public WebDriver driver;
+    public WebDriverWait wait;
```

Comment:
I will add a new method on my next commit I am refactoring and converting
this to abstract class.
Sorry

2015-08-30 20:49 GMT-04:00 Carledriss Zeiss <notifications@github.com>:

> In src/main/java/com/selenium/course/pages/Page.java
> <https://github.com/SeleniumCourse/SFDC-Team4/pull/18#discussion_r38278993>
> :
>
> >  public class Page {
> > -    protected WebDriver driver;
> > -    protected WebDriverWait wait;
> > +    public WebDriver driver;
> > +    public WebDriverWait wait;
>
> Why these attributes are public?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SeleniumCourse/SFDC-Team4/pull/18/files#r38278993>.
>